xamarin/Xamarin.Forms
# [UWP] Fixes jumps at scroll view

### Description of Change ###

Because changes from #4703 leads to issue #5652, I've removed interface `IDontGetFocus`. However I think that problem is deeper - method [`OnGotFocus`](https://github.com/xamarin/Xamarin.Forms/blob/24bf23b2d1060be787d8a9ed4c1626f7a2165ebe/Xamarin.Forms.Platform.UAP/VisualElementRenderer.cs#L177) should respect area of `ScrollView` and shouldn't allow inner elements being focused by `FocusManager.TryMoveFocus`.

### Issues Resolved ### 


- fixes #5652

### API Changes ###

 
 None

### Platforms Affected ### 


- UWP

### Behavioral/Visual Changes ###


None

### Before/After Screenshots ### 


Not applicable

### Testing Procedure ###


- Open page Issue5652
- Scroll down to the button with caption "Click around"
- Click somewhere near button
- Label with placeholder "Should not jump here" will not gain focus, thus no scrollview jumping occurred.

### PR Checklist ###
